kolibri/deployment/default/settings/base.py

```diff
@@ -86,6 +87,10 @@
     }
 }
 
+# DIR for storing contentDBs
+CONTENT_DB_DIR = os.path.abspath(os.path.join(__file__, "../../../..")) + "/content/content_db"
+# DIR for storing content copies for all channels
+CONTENT_COPY_DIR = os.path.abspath(os.path.join(__file__, "../../../..")) + "/content/content_copy"
```

Comment:
currently, the KOLIBRI_HOME is auto set by the application. Will it be better to ask the user to set the KOLIBRI_HOME at first time installation?